subsys/usb/device_next/class/usbd_cdc_acm.c

```diff
@@ -117,6 +117,8 @@ struct cdc_acm_uart_data {
 	struct k_work rx_fifo_work;
 	atomic_t state;
 	struct k_sem notif_sem;
+	/* ZLP needed if no more tx is needed after this tx is done */
+	bool zlp_needed;
```

Comment:
You can move it below `bool flow_ctrl;` to save some RAM on padding.

Reply:
Also `bool flow_ctrl;` could be moved around, so all the bool variables (`line_state_rts`, `line_state_dtr`, `flow_ctrl`, `zlp_needed`) would be next to each other, which would save few more bytes.

Reply:
Hi @tmon-nordic, I think that is a good point on the packing overhead. by moving both `bool flow_ctrl` and `bool zlp_needed` we could save 8 bytes of RAM, by moving them t the original padding space of `line_state_rts`, `line_state_dtr`. 
I have made the change as suggested.